spec/Section 6 -- Execution.md

```diff
@@ -1,6 +1,6 @@
 # Execution
 
-GraphQL generates a response from a request via execution.
+A GraphQL service generates a response from a request via execution.
```

Comment:
Similar to above, this could be “A GraohQL Executor,,,”

Reply:
But GraphQL service is in many more places used... also this should be a separate PR.

Reply:
I think service is correct in both cases. A GraphQL service does both validation and execution.

Reply:
See above, should be separate PR if at all controversial

Reply:
I suggest to put changing to executor here and elsewhere (if we decide to) into follow up PR. Executor is used in 11 places, but it is never explained what is that. Change from 'GraphQL' to 'GraphQL service' is improvement, as GraphQL is an abstract thing, it cannot ACT, it cannot do anything.